posix.mak

```diff
@@ -378,7 +378,7 @@ style_lint:
 auto-tester-build: target checkwhitespace
 
 .PHONY : auto-tester-test
-auto-tester-test: unittest benchmark-compile-only
+auto-tester-test: unittest test_all benchmark-compile-only
```

Comment:
Tests fail because there is no `test_all` target in posix.mak. The "additional tests" are run as dependency of `unittest`.